src/database/sql/sequelize/models/biometrics/blood.oxygen.saturation.model.ts

```diff
@@ -38,71 +36,40 @@ export default class BloodOxygenSaturation extends Model {
     })
     id: string;
 
-    @Length({ min: 2, max: 128 })
-    @Column({
-        type      : DataType.STRING(128),
-        allowNull : true,
-    })
-    EhrId: string;
-
-    @IsUUID(4)
-    @ForeignKey(() => Person)
-    @Column({
-        type      : DataType.UUID,
-        allowNull : true,
-    })
-    PersonId: string;
-
-    @BelongsTo(() => Person)
-    Person: Person;
-
     @IsUUID(4)
     @ForeignKey(() => User)
     @Column({
         type      : DataType.UUID,
-        allowNull : true,
+        allowNull : false,
     })
     PatientUserId: string;
 
-    // @IsUUID(4)
-    // @ForeignKey(() => Encounter)
-    // @Column({
-    //     type      : DataType.UUID,
-    //     allowNull : true,
-    // })
-    // EncounterId: string;
-
-    @IsDecimal
+    @Length({ max: 64 })
     @Column({
-        type      : DataType.FLOAT,
+        type      : DataType.INTEGER,
```

Comment:
FLOAT was correct datatype. Please remove @Length({ max: 64 }) and add back @IsDecimal.
Validation 
```
validate  : {
            min : 10,
            max : 100
        }
```
was also correct. Please revert that also.